web/repos.jspf

```diff
@@ -171,8 +172,13 @@ Copyright (c) 2015, 2016, Oracle and/or its affiliates. All rights reserved.
                             if (branch == null) {
                                 branch = "N/A";
                             }
+                            String currentVersion = ri.getCurrentVersion();
+                            if (currentVersion == null) {
+                                currentVersion = "N/A";
+                            }
                             %><td><%= Util.htmlize(type) %></td><%
                             %><td><%= Util.htmlize(parent) %> (<%= Util.htmlize(branch) %>)</td><%
+                            %><td><%= Util.htmlize(currentVersion) %> </td><%
```

Comment:
My output does not match your output. Especially I don't like the enclosing \'.

![screenshot from 2016-11-03 15-36-27](https://cloud.githubusercontent.com/assets/6997160/19970503/70bb94a6-a1dc-11e6-9772-4dfdb2616ffa.png)